addons/l10n_es/data/account_fiscal_position_template_data.xml

```diff
@@ -490,43 +490,43 @@
             model="account.fiscal.position.tax.template">
             <field name="position_id" ref="fp_not_subject_tai"/>
             <field name="tax_src_id" ref="account_tax_template_s_iva4b"/>
-            <field name="tax_dest_id" ref="account_tax_template_s_iva_ns_b"/>
+            <field name="tax_dest_id" ref="account_tax_template_s_iva0_e"/>
         </record>
         <record id="fptt_not_subject_tai_ventas_4s"
             model="account.fiscal.position.tax.template">
             <field name="position_id" ref="fp_not_subject_tai"/>
             <field name="tax_src_id" ref="account_tax_template_s_iva4s"/>
-            <field name="tax_dest_id" ref="account_tax_template_s_iva_ns"/>
+            <field name="tax_dest_id" ref="account_tax_template_s_iva0_e"/>
         </record>
         <record id="fptt_not_subject_tai_ventas_10b"
             model="account.fiscal.position.tax.template">
             <field name="position_id" ref="fp_not_subject_tai"/>
             <field name="tax_src_id" ref="account_tax_template_s_iva10b"/>
-            <field name="tax_dest_id" ref="account_tax_template_s_iva_ns_b"/>
+            <field name="tax_dest_id" ref="account_tax_template_s_iva0_e"/>
```

Comment:
This one should be changed, as we are talking about goods. Isn't it?

Reply:
Tax exemption on Spain can be referred to the following articles of https://boe.es/buscar/act.php?id=BOE-A-1992-28740:

- Article 20: Mainly for healthcare, insurances and education. On 303 report, should go to 120 box
- Article 21, 22, 23 & 24: Exportations and similars. On 303 report, should go to 60 box
- Article 25: EU exportations. On 303 report, should go to 59 box

According to description of the modified Fiscal Position, we are referring to:
"Está posición fiscal solo aplica en el caso de vender desde territorio nacional a una localización no sujeta, por ejemplo, Canarias, Ceuta o Melilla.". That can be translated as "This fiscal position is applied in the case of selling from national territory to a non subject localization, like Canary Island, Ceuta or Melilla". 

Canary Islands, Ceuta and Melilla are considered outside of the IVA application law, so it must considered as exportation.

So, we should modify the description of the Fiscal Position if you are expecting your proposal. However, you should remember that article 20 is not including sales of goods in most of the cases. Also, the people exempt are paying VAT on their purchases. For example:

A hospital is selling healthcare services with VAT exemption. However, if they sell you a crutch, they must sell it with VAT applied. If they buy a medication, they will buy it with VAT applied, but, as they are not selling the medication but the administration of medication, they will sell it without VAT.

So, this case applies to the good not the customer and should be applied as a VAT Exempt on the product